packages/ozone/src/config/config.ts

```diff
@@ -67,6 +74,11 @@ export type ServiceConfig = {
   version?: string
 }
 
+export type BlobReportServiceConfig = {
+  url?: string
+  authToken?: string
+}
```

Comment:
It looks like both `url` and `authToken` are required in order to use the blob report service.  Could we make these both required here, and update `OzoneConfig` to allow it not to be configured:
```ts
export type OzoneConfig = {
  // ...
  blobReportService: BlobReportServiceConfig | null
}
```